sources/packages/backend/apps/api/src/route-controllers/application-exception/_tests_/application-exception-helper.ts

```diff
@@ -0,0 +1,53 @@
+import { TestingModule } from "@nestjs/testing";
+import { SystemUsersService } from "@sims/services";
+import {
+  Application,
+  ApplicationException,
+  ApplicationExceptionStatus,
+  Student,
+} from "@sims/sims-db";
+import {
+  createFakeApplication,
+  createFakeApplicationException,
+  createFakeStudent,
+} from "@sims/test-utils";
+import { AESTGroups, getAESTUser } from "../../../testHelpers";
+import { DataSource } from "typeorm";
+
+/**
+ * Create a fake application with an application exception associated.
+ * @param applicationExceptionStatus application exception status.
+ * @param dataSource application dataSource.
+ * @param module application module.
+ * @returns application with an application exception associated.
+ */
+export async function saveFakeApplicationWithApplicationException(
+  applicationExceptionStatus: ApplicationExceptionStatus,
```

Comment:
IMO applicationExceptionStatus and systemUser(mentioned in the below comment) can be passed inside `options:{applicationExceptionStatus? :ApplicationExceptionStatus,systemUserId:?number }`

Reply:
removed the systemUser. Thanks!